### ISSUE TITLE
add a technical doc about how Milestones are generate and validated

### ISSUE THREAD

**Issue by alon-e** [iotaledger/compass#15]
"We now need to bootstrap the coordinator milestone merkle tree." 
in the "How To", lacks an explanation what it is, what is it used for etc.

**Comment by jakubcech** [comment]
@JakeSCahill do we have anything on this specifically? On the relationships between this and the milestones other than what is here? :) 
https://docs.iota.org/docs/compass/0.1/how-to-guides/create-an-iota-network?q=merkle%20tree&highlights=sleep;seed;press;prefer;order;need;feel;free;creat;break;rest;refer;token;test;trust;tree#calculate-the-merkle-tree

**Comment by JakeSCahill** [comment]
@jakubcech Yes 😄 In that article, the link on 'Merkle tree' goes here: https://docs.iota.org/docs/the-tangle/0.1/concepts/the-coordinator#milestones
